src/Hacknet/ui/HacknetServerElem.tsx

```diff
@@ -60,15 +60,18 @@ export function HacknetServerElem(props: IProps): React.ReactElement {
       multiplier = Math.min(levelsToMax, purchaseMult as number);
     }
 
-    const increase =
+    const base_increase =
       calculateHashGainRate(node.level + multiplier, 0, node.maxRam, node.cores, props.player.hacknet_node_money_mult) -
-      node.hashRate;
+      calculateHashGainRate(node.level, 0, node.maxRam, node.cores, props.player.hacknet_node_money_mult);
+    const modded_increase = (base_increase * (node.maxRam - node.ramUsed)) / node.maxRam;
+
     const upgradeLevelCost = node.calculateLevelUpgradeCost(multiplier, props.player.hacknet_node_level_cost_mult);
     upgradeLevelButton = (
       <Tooltip
         title={
           <Typography>
-            +<HashRate hashes={increase} />
+            +<HashRate hashes={base_increase} /> (base increase, attained when no script is running) <br />
```

Comment:
```suggestion
            <span style={{ opacity: 0.75 }}>
              +<HashRate hashes={base_increase} /> (base increase, attained when no script is running) <br />
            </span>
```
maybe change lower the opacity of the "inactive" rates by doing something to this effect? so that the actual rate stands out more